stack-definition.yaml

```diff
@@ -461,18 +461,14 @@ components:
 
 changeLog: |
 
-  - 🐛 Fixing Grafana dashboards that were broken since Gimlet Stack v0.4.0 aka the "2021.06-2" update
 
-  - 🍏 Ingress Nginx from version 0.47.0 to 0.49
-    Minor improvments in this one.
-
-    ❗ Ingress Nginx v1 is released, and in the next Gimlet Stack update we are going to upgrade to that.
+    ❗ We are investigating Ingress Nginx. There is upcoming changes with v1 (it is released), and in the next Gimlet Stack update we are going to upgrade to that.
     Please be prepared as it only supports Kubernetes versions >= v1.19
 
-  - 🍏 Cert Manager update to 1.5.3 from 1.4.0
+  - 🍏 Grafana update from version 6.13.5 to 6.16.9
     Just a regular cadence of version updates; including fixes, features, etc
 
-  - 🍏 Prometheus is updated to use kube-state-metrics 2.1.0 instead of 2.0.0
+  - 🍏 Prometheus update from version 14.6.0 to 14.7.1
```

Comment:
Same for Prometheus version.
If you link to the Prom release log, with a summary of what's in there, like "mostly bugfixes, and minor improvements" is good enough.